gateway/src/apicast/configuration.lua

```diff
@@ -114,6 +114,8 @@ function _M.parse_service(service)
       backend = backend,
       oidc = {
         issuer_endpoint = value(proxy.oidc_issuer_endpoint),
+        client_id = value(proxy.oidc_client_id),
```

Comment:
nitpick: the name of this var suggests that this is the client ID itself, when in reality, it's the jwt claim that contains the client ID. Can we rename this to `jwt_claim_with_client_id` or similar ?